### PR TITLE
testing/radicale: modernize

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

**Comment by ncopa** [comment]
whoops, i updated this in parallel. sorry.

Please test whats in `edge/testing`. I had to clean up the `py-vobject` deps.

**Comment by TBK** [comment]
@ncopa No problem. I have changed this PR so it only encompasses the additional changes I made to the APKBUILD file.

File: testing/radicale/APKBUILD [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
-depends_dev=""
-makedepends="$depends_dev python3"
+makedepends="python3"
 pkgusers="radicale"
 pkggroups="radicale"
 install="$pkgname.pre-install"
-subpackages=""
-source="https://files.pythonhosted.org/packages/source/R/Radicale/Radicale-$pkgver.tar.gz
+options="!check" # no test suite
+source="https://files.pythonhosted.org/packages/source/R/$_pkgname/$_pkgname-$pkgver.tar.gz
```
Comment (ncopa) [inline testing/radicale/APKBUILD]: I prefer not use `$_pkgname` at all. It makes copy/paste the source URL to web browser harder.

**Comment by TBK** [comment]
@ncopa Fixed :)

**Pull request closed without merging**